docker-entrypoint.sh

```diff
@@ -3,45 +3,49 @@ set -e
 
 ADMIN_USER=${ADMIN_USERS%%,*}
 ADMIN_PASSWORD=${ADMIN_PASSWORD:-alerta}
+MAXAGE=${ADMIN_KEY_MAXAGE:-315360000}  # default=10 years
 
 # Generate minimal server config, if not supplied
 if [ ! -f "${ALERTA_SVR_CONF_FILE}" ]; then
+  echo "# Create server configuration file."
   cat >"${ALERTA_SVR_CONF_FILE}" << EOF
 SECRET_KEY = '$(< /dev/urandom tr -dc A-Za-z0-9_\!\@\#\$\%\^\&\*\(\)-+= | head -c 32)'
 EOF
 fi
 
+# Init admin users and API keys
+if [ -n "${ADMIN_USERS}" ]; then
+  echo "# Create admin users."
+  alertad user --all --password "${ADMIN_PASSWORD}"
```

Comment:
Before this commit, this line had `|| true` on the end - but now if I am using an 'openid' $AUTH_PROVIDER and have $ADMIN_USERS defined this change breaks that configuration. Is that intended?